.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  repository_dispatch:
+    types: [patch-release]
+
+jobs:
+  patch-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine release branch
+        id: branch_step
+        run: |
+          branch=release-$(echo ${{ github.event.client_payload.version }} | cut -d '.' -f 1,2).x
+          echo "Derived release branch: $branch (this branch must already exist for patch releases)"
+          echo "##[set-output name=branch;]$branch"
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.branch_step.outputs.branch }}
+      - name: Fetch tags
+        run: |
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - name: Validate tag
+        run: |
+          git tag v${{ github.event.client_payload.version }}
+      - name: Update manifests version
+        run: |
+          cat deploy/operator/basic/deployment.yaml | sed "s|image: armory/spinnaker-operator:.*|image: armory/spinnaker-operator:${{ github.event.client_payload.version }}|" | sed "s|imagePullPolicy:.*|imagePullPolicy: IfNotPresent|" > deploy/operator/basic/deployment.yaml.new
```

Comment:
not `always`?

Reply:
oh well.... i guess since it's tagged, it should never change. OK!

Reply:
`Always` is used for dev work because we can update the same tag (`dev`) over and over. For releases we use unique tags for each release, so `imagePullPolicy` can be changed to `IfNotPresent`.